core/src/components/list/list.md.scss

```diff
@@ -68,9 +62,22 @@
 // Material Design Inset Lines List
 // --------------------------------------------------
 
-.list-md-lines-inset .item,
+/**
+ * We remove the full lines below, but developers
+ * should still be able to override the lines on
+ * a per-item basis. As a result, we this selector
+ * to exclude a) items with full lines, b) items with
+ * no lines, and c) the last item in an inset list.
+ */
+.list-md-lines-inset .item:not(.item-lines-full):not(.item-lines-none):not(:last-child),
```

Comment:
This gnarly selector is so you can still override lines on a per-item basis. Any other ideas for how to do this? I know you can do complex selectors in a single `:not`  but that is not available in all browser versions we support.